index.html

```diff
@@ -64,31 +64,21 @@
         ],
 
         // name of the WG
-        wg:           "W3C Credentials Community Group",
+        wg:           "Verifiable Claims Working Group",
 
         // URI of the public WG page
-        wgURI:        "http://www.w3.org/community/credentials/",
+        wgURI:        "https://www.w3.org/2017/vc/",
 
         // name (with the @w3c.org) of the public mailing to which comments are due
-        wgPublicList: "public-credentials",
+        // wgPublicList: "public-vc-comments",
 
         // URI of the patent status for this WG, for Rec-track documents
         // !!!! IMPORTANT !!!!
         // This is important for Rec-track documents, do not copy a patent URI from a random
         // document unless you know what you're doing. If in doubt ask your friendly neighbourhood
         // Team Contact.
-        wgPatentURI:  "",
+        wgPatentURI:  "https://www.w3.org/2004/01/pp-impl/98922/status",
         maxTocLevel: 4,
-        otherLinks: [{
-          key: "Version Control",
-          data: [{
-            value: "Github Repository",
-            href: "https://github.com/opencreds/vc-data-model"
-          }, {
-            value: "Issue Tracker",
-            href: "https://github.com/opencreds/vc-data-model/issues"
-          }]
-        }],
```

Comment:
Are these links auto-generated because you set 'github' above? If so, +1... if not, we should make sure folks can still get to the repo and issue tracker via links at the top of the spec. I'm assuming it's the former.